### PR TITLE
Milestone 2: Finish mobile version

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
# Porfolio Milestone 2: Finish mobile version

## Changes made in this milestone:
- Added work and about sections 
- Updated style.css

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @HenaryKhuraijam , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌

- [ ] Please use all sections of the [README](https://github.com/microverseinc/readme-template) file 👎🏾 
- [ ] Your live demo link is broken, please fix this too. 👎🏾 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by HenaryKhuraijam** [comment]
Make changes as suggested.

File: index.html [hunk 1]
```diff
@@ -10,36 +10,210 @@
...
+            <li class="social-items">
+              <a href="https://angel.co/">
+                <img src="images/angellist.png" alt="angellist link">
+              </a>
+            </li>
+          </ul>
+        </div> 
+      </div>   
+    </section>
+    <section class="works" id="works">
```
Comment (NduatiKagiri) [inline index.html:68]: - [ ] You have done a commendable job implementing the desired requirements 💯, but kindly make sure you apply the right background color to your Works section.
-   Your design:
    henry1

-   Figma design:
    henry2

-   Please check the background color(on the areas I have marked) in the design, and apply it to your design.

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @HenaryKhuraijam ,

# CHANGES REQUIRED ♻️  

Good job so far!
There is one issue that you still need to work on to go to the next project but you are almost there!

so close

## Highlights ✔️ 
✅ GitHub Flow observed
✅ No Linters errors
✅ Good/smooth UI

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HenaryKhuraijam** [comment]
Make changes as suggested.

**Review by victorokoroji — APPROVED** [review]
Hi @HenaryKhuraijam, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**